public/javascripts/index.js

```diff
@@ -6,52 +6,59 @@ function scrollToElement (element) {
 }
 
 function renderError (el, message) {
-  console.log('Appending: ' + message)
-  el.text(' - ' + message)
+  console.log(message)
```

Comment:
since that we have winston log, should we just use that (`logger.x`) and not `console.log`?